submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,161 @@
+const NUM_OF_FRIENDS = 20;
+const FRIENDS = document.querySelector(".friends");
+let allFriends;
+let friendsForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function fetchFriends() {
+  const api_url = `https://randomuser.me/api/?results=${NUM_OF_FRIENDS}`;
+  fetch(api_url)
+    .then(handleErrors)
+    .then((response) => response.json())
+    .then((data) => (allFriends = data.results))
+    .then(() => addFriends(allFriends))
+    .catch((error) => showErrorMessage());
+}
+
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+function showErrorMessage() {
+  const wholePage = document.querySelector("body");
+  wholePage.setAttribute("class", "refresh");
```

Comment:
Wuuuuuut? O_o Why you are using `setAttribute` to change classname?

Reply:
Got it. Add class via `innerHtml`